Makefile

```diff
@@ -1,3 +1,10 @@
+ifeq ($(OS),Windows_NT)
+SEPARATOR := ;
+else
+SEPARATOR := :
+endif
+
+
```

Comment:
Potentially stupid question ahead: 
What is the benefit of using `$(SEPARATOR)` in the first place? Is this so that we can use a different separator per system? ( I think I read that windows uses `;` for instance)?
Or is there something beyond this?